charts/kube-state/values.yaml

```diff
@@ -32,3 +32,35 @@ replicas: 1
 
 extraArgs:
 - "--metric-labels-allowlist=namespaces=[*],pods=[*]"
+
+priorityClassName: ~
+
+collectors:
```

Comment:
TODO: Remove unused collectors and test metrics working without disruption

Reply:
i have commented out the unused collectors